site/content/ytt/docs/latest/how-to-export-schema.md

```diff
@@ -0,0 +1,159 @@
+      properties:
+        load_balancer:
+          type: object
+          additionalProperties: false
+          properties:
+            enabled:
+              type: boolean
+              default: true
+            static_ip:
+              type: string
+              default: null
+              nullable: true
+        app_domains:
+          type: array
+          items:
+            type: string
+            default: ""
+          default:
+          - apps.example.com
+          - mobile.example.com
+        databases:
+          type: array
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              name:
+                type: string
+                default: ""
+              adapter:
+                type: string
+                default: postgresql
+              host:
+                type: string
+                default: ""
+              port:
+                type: integer
+                default: 5432
+          default: []
+        additional_config:
+          nullable: true
+          default: {}
+```
+
+## Exported Properties
+
+Of the properties declared in the [OpenAPI Schema specification](https://swagger.io/specification/#schema-object), the following are generated.
+
+### `type`
+
+Names the type of the schema.
+
+The value of this property depends on the type of data value implied in the `ytt` schema:
+- map ==> `object`
+- array ==> `array`
+- boolean ==> `bool`
+- string ==> `string`
+- integer ==> `integer`
+- float ==> `type: number; format: float`
+
+### `default`
+
+Declares a default value.
+
+The value of this property:
+- is always included;
+- is typically the value of the data value in the `ytt` schema;
+- if the data value is annotated `@schema/default`, _that_ value is used instead.
+
+### `additionalProperties`
+
+Indicates whether other keys are allowed in a mapping/object.
+
+- whenever inspecting a data value that is a map, `ytt` includes this property, setting it to `false`;
+- when a data value is annotated `@schema/type any=True`, this property is omitted.
+
+### `nullable`
+
+Indicates whether `null` is also allowed.
+
+- when a data value is annotated `@schema/nullable` or `@schema/type any=True`, this property is included and set to `true`;
+- otherwise, this property is omitted.
+
+### `description`
+
+Explains the contents and/or consequences of certain values of the property.
+
+- when a data value is annotated `@schema/desc`, the value of that description is the value of this property, verbatim;
+- otherwise, this property is omitted.
+
+## Known Limitations
+
+The following are not yet supported in `ytt`:
+- `ytt` schema does not yet support declarative validations and thus does not produce such validations in an OpenAPI export.
+- there is no means yet to modify the [`info` section of the OpenAPI document](https://swagger.io/specification/#info-object) from within a `--data-values-schema-inspect`.
```

Comment:
```suggestion
- there is no means yet to modify the [`info` section of the OpenAPI document](https://swagger.io/specification/#info-object) from within a `--data-values-schema-inspect`. There are defaults supplied, however we recommend that these fields are updated manually.
```

Reply:
> Document how to insert an OpenAPI components.schemas.dataValues into a kapp-controller Package (maybe in the kapp-controller docs?)

I think I failed to ping you on that PR: #301. I just do so, now.

Reply:
> add a @schema/desc annotation example somewhere so its easier to notice.

Love this and will...

Reply:
Done.